File: Selenium/Activities/Activity12.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity12 {

    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Title: " + driver.getTitle());
        WebElement text=driver.findElement(By.id("input-text"));
        System.out.println("Before: "+text.isEnabled());
        driver.findElement(By.id("toggleInput")).click();
        System.out.println("After: "+text.isEnabled());
        driver.close();

    }

    }

File: Selenium/Activities/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("Title: " + driver.title)

    text=driver.find_element(By.ID,"input-text")
    print("Before: ",text.is_enabled())
    driver.find_element(By.ID,"toggleInput").click()
    print("After: ",text.is_enabled())
    driver.close()

